packages/suite/src/support/messages.ts

```diff
@@ -2945,6 +2945,10 @@ export default defineMessages({
         defaultMessage: 'Legacy accounts',
         id: 'TR_LEGACY_ACCOUNTS',
     },
+    TR_LEGACY: {
```

Comment:
There is already the same translation string for `TR_ACCOUNT_TYPE_BIP44_NAME` so maybe no need to introduce it or delete the old one and keep this one.

Reply:
I think this is just some WIP POC

Reply:
`TR_${accountType.toUpperCase()}_ACCOUNT`